spinedb_api/db_mapping_base.py

```diff
@@ -323,6 +358,8 @@ def __init__(self, db_map, item_type, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._db_map = db_map
         self._item_type = item_type
+        self._id_factory = IdFactory()
+        self.id_map = IdMap()
         self._id_by_unique_key_value = {}
         self._temp_id_by_db_id = {}
```

Comment:
The self._temp_id_by_db_id is not used anymore it seems.

Reply:
Indeed. I will remove what is left of it.